client/bundles/Main/components/sessions/AllSessions.jsx

```diff
@@ -7,7 +7,7 @@ const AllSessions = props => {
 	let sessions = sessionlist.map(session => {
 
 		return (
-			<div className="wrapper">
+			<div className="wrapper" key={session.id} >
```

Comment:
Why you added this key?

Reply:
if we don't have the ke we get the folowing error "Each child in an array or iterator should have a unique "key" prop."